developer_documentation/data_format.rst

```diff
@@ -1743,20 +1743,23 @@ Offset  Length  Version  Field name          Description
  216       4      9-     message_guest       Inbox: Guest Window button.
  220       4      9-     message_ride        Inbox: Ride Instance Window button.
  224       4      9-     message_ride_type   Inbox: Ride Select GUI button.
- 228       4     10-     toolbar_main        Toolbar: Main menu GUI button.
- 232       4     10-     toolbar_speed       Toolbar: Speed menu GUI button.
- 236       4     10-     toolbar_path        Toolbar: Path build GUI button.
- 240       4     10-     toolbar_ride        Toolbar: Ride Select GUI button.
- 244       4     10-     toolbar_fence       Toolbar: Fence build GUI button.
- 248       4     10-     toolbar_scenery     Toolbar: Scenery GUI button.
- 252       4     10-     toolbar_terrain     Toolbar: Terraform GUI button.
- 256       4     10-     toolbar_staff       Toolbar: Staff GUI button.
- 260       4     10-     toolbar_inbox       Toolbar: Inbox GUI button.
- 264       4     10-     toolbar_finances    Toolbar: Finances GUI button.
- 268       4     10-     toolbar_objects     Toolbar: Path objects GUI button.
- 272       4     10-     toolbar_view        Toolbar: View menu GUI button.
- 276       4      2-     texts               Text of the guis (reference to a TEXT block).
- 280                                         Total length.
+ 224       4     12-     loadsave_err        Loadsave icon for files with errors.
```

Comment:
Above you don't delete the line with offset 224, so now two variables live at the same offset.

Reply:
Not sure why you don't simply add it to the bottom, but fair enough.